### ISSUE TITLE
Broken css with FireFox

### ISSUE THREAD

**Issue by alive-corpse** [AnthonMS/ha-slider-card#5]
Hi! That's how plugin looks at firefox. Any changes in options take effect only for background color.
Default settings:
![image](https://user-images.githubusercontent.com/3852606/121932375-400ac680-cd5e-11eb-9cbd-bca5f8ab08f0.png)

Custom config:
```
type: 'custom:slider-card'
entity: light.lamp02_light
height: '30px'
mainSliderColor: 'green'
secondarySliderColor: 'red'
thumbWidth: '0px'
thumbHeight: '30px'
thumbColor: 'pink'
thumbHorizontalPadding: '0px'
thumbVerticalPadding: '0px'
```
![image](https://user-images.githubusercontent.com/3852606/121932727-a394f400-cd5e-11eb-908f-fbf4b218b215.png)

**Comment by AnthonMS** [comment]
Sorry about that. I don't know why it looks like that on Firefox. I have been using brave and I'm pretty sure that's based on Firefox. 

I don't know if I will get any time to look at it in the near future. Haven't even looked at my HA in a couple months... You are more than welcome to try and fix it yourself, and make a pull request with the changes.

**Comment by AnthonMS** [comment]
It should be fixed by @Garulf in PR #17 

Let us know if the bug persists.

Garulf's comment: 
> Go for it, he may have to adjust his styling as firefox is a little wacky on thumb dimensions
